ext/systemd/puppet.service

```diff
@@ -6,6 +6,7 @@ After=basic.target network.target puppetmaster.service
 [Service]
 EnvironmentFile=-/etc/sysconfig/puppetagent
 EnvironmentFile=-/etc/sysconfig/puppet
+EnvironmentFile=-/etc/default/puppet
```

Comment:
We could template it during build using sed (that's what works right now), but that leaves the file basically useless to anyone else because we'd need to use some sort of token in place of the environment file.